### DISCUSSION TITLE
Enable drill-down analysis / fix panel linking

### DISCUSSION THREAD

**Discussion by lucaswhornsby** [grafana/grafana#55219] [Dashboards, Panels & Visualizations]
Currently, we are using the Datalink option to link the panels on a particular dashboard (for orchestrator pipeline and trigger status). However, we are having trouble filtering some of the dimensions by clicking on them in the table. When we click on the desired dimension (e.g., a particular TriggerID), the filter is not being applied because the TriggerID is not being passed (and we can tell in the URL it is remaining static), although it’s supposed to update to a new __fieldname for the chosen variable/dimension. The same applies for TriggerName, which only works when we select a TriggerName from the dropdown menu and not when we click on it.

**Please see the attached screen recording for a demonstration of the problem.** 

**Environment**:
- Grafana version: Azure Managed Grafana 
- Data source type & version: Data in this particular dashboard is coming from Azure Data Factory
- OS Grafana is installed on: Windows 11
- User OS & Browser: Edge

[Grafana Panel Linking Error.webm](https://user-images.githubusercontent.com/113460677/189957410-2c53749a-cedb-49ee-98b9-fd74c0a72761.webm)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
